src/server.js

```diff
@@ -17,12 +19,27 @@ app.use(cors());
 
 app.use(morgan("combined"));
 
+// Used by certain endpoints for verification
+async function verify(token, email) {
+  const ticket = await client.verifyIdToken({
```

Comment:
what's going to happen if this call fails?

Reply:
what do you mean when call fails? Like invalid input or an error occurs within this function?

Reply:
What I mean is, what if your connection to client is not stable? What if google server is down and this verifyIdToken never returns?

Reply:
I will wrap this function call within a try catch each time it is called within the endpoints and if any error occurs, then return an error status and message; otherwise proceed as normal